app/src/main/java/com/example/paetz/yacguide/RouteActivity.java

```diff
@@ -115,12 +118,20 @@ public void onClick(View v) {
                 typeface = Typeface.ITALIC;
                 bgColor = Color.LTGRAY;
             }
+
+            final Ascend[] ascends = db.ascendDao().getAscendsForRoute(route.getId());
+
+            int lowestAscendId = 14;
+            for (final Ascend ascend: ascends) {
+                lowestAscendId = min(lowestAscendId, ascend.getStyleId());
+            }
+
             layout.addView(WidgetUtils.createCommonRowLayout(this,
-                    route.getName() + commentCountAddon,
+                    WidgetUtils.getAscendSymbol(lowestAscendId) + " " + route.getName() + commentCountAddon,
```

Comment:
Imagine you have a follow-ascend and a lead-"Sack" at the same route. --> Multiple symbols must be possible.

Reply:
The information about ascended / sacked / watched / projected should be stored in the db field in Route table (can we hack field "ascendCount" for that?). Doing so, we prevent iterating over all ascends each time we enter a route list.

Reply:
I thought about that, but my knowledge about the framework is very limited and I did not want to break the database compatibility.